src/app/pages/Details/Details.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import BackButton from "../../components/BackButton/BackButton";
+import FavoriteCheckbox from "../../components/FavoriteCheckbox/FavoriteCheckbox";
+import IconButton from "../../components/IconButton/IconButton";
+import FavoriteIcon from "../../components/Icons/FavoriteIcon";
+import RequestIcon from "../../components/Icons/RequestIcon";
+import styles from "./Details.module.css";
+
+export default function Details(): JSX.Element {
+  return (
+    <div className={styles.container}>
+      <header className={styles.header}>
+        <BackButton />
+      </header>
+      <article className={styles.hutaDetails__card}>
+        <div className={styles.hutaDetails__card_header}>
+          <FavoriteCheckbox icon={<FavoriteIcon />} value="favorite" />
+          <h3 className={styles.hutaDetails__card_title}>Dog's Paradise</h3>
+          <img
+            className={styles.hutaDetails__img}
+            src="src/app/assets/images/huta_example1.png"
+            alt=""
+          />
+        </div>
+        <IconButton title="Send request">{<RequestIcon />}</IconButton>
+        <p className={styles.hutaDetails__info}>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
```

Comment:
Maybe use a placeholder text that fits the topic of your app a bit better? 😊

Reply:
Yes, you're absolutely right.